.yamato/training-int-tests.yml

```diff
@@ -1,7 +1,5 @@
 test_editors:
   - version: 2018.4
```

Comment:
Can you add 2020.1 to standalone-build-test.yml, add it back here, and add 2019.3 here?